t/lib/OpenQA/Test/Utils.pm

```diff
@@ -131,7 +131,7 @@ sub fake_asset_server {
             $c->render(text => 'This file was not compressed!', format => 'txt');
         });
     $r->get(
-        '/tests/:job/asset/:type/:filename' => sub {
+        '/tests/:job/asset/:type/:filename' => [format => ['qcow2']] => sub {
```

Comment:
This looks wrong to me. There's no content negotiation here that would need a format. It was probably kinda broken before already, and the `:filename` should have been a `*filename`, to capture the whole filename.

Reply:
Well, I was just following the guide in the changelog, which didn't mention that being a possibility. It clearly broke because of the change, and this was what the changelog suggested doing to fix it, and it fixed it, so...

Reply:
changed, though.